contracts/src/SubstrateTypes.sol

```diff
@@ -56,18 +56,13 @@ library SubstrateTypes {
      * `NativeTokensMessage::Create`
      */
     // solhint-disable-next-line func-name-mixedcase
-    function RegisterToken(address gateway, address token, bytes2 createCallIndex)
-        internal
-        view
-        returns (bytes memory)
-    {
```

Comment:
Please don't make any formatting/whitespace changes. If necessary, we should disable any auto-formatters until the audit is complete.

This will help the auditors.

Reply:
This is a relevent change for remove `createCallIndex` which shorten code line and triggers the format. 

I would assume this is revelant to intention of the fix and keep it, or we may need to disable the rule [here](https://github.com/Snowfork/snowbridge/blob/4f739fde129de640c667dc10eb0a08b701183bdc/hooks/pre-commit#L23).